src/shim/sdk/GroupSDK.ts

```diff
@@ -35,7 +35,7 @@ export function get(groupID: string) {
  * Create a new group. Takes an options object which allow for specifying an optional, unencrypted ID and name for the group.
  * @param {GroupCreateOptions} options Group creation options
  */
-export function create(options: GroupCreateOptions = {groupName: "", addAsMember: true, needsRotation: false}) {
+export function create(options: GroupCreateOptions = {groupName: "", addAsMember: true, addAsAdmin: true, needsRotation: false}) {
```

Comment:
```suggestion
export function create(options: GroupCreateOptions = {groupName: "", addAsMember: true, addAsAdmin: true, owner: "", needsRotation: false}) {
```

Reply:
I don't think we wan this to default to an empty string. If an owner is not specified undefined is passed to the server and the creator is made the owner by default(as long as addAsAdmin it true).

Reply:
I don't think that's true based on the code you have below. If this isn't provided it'll be undefined but it will then be converted back to an empty string below in the `ownerUserId: options.ownerUserId || ""`. I don't think it matters all that much either way really, just pointing that out.

Reply:
I see what you are saying. I took off the || "".